calico/getting-started/kubernetes/requirements.md

```diff
@@ -57,6 +57,4 @@ Note that Kubernetes version 1.16+ requires Istio version 1.2 or greater.
 Note that Istio version 1.9 requires Kubernetes version 1.17-1.20.
 Note that Istio version 1.10 is supported on Kubernetes version 1.18-1.21, but has been tested on Kubernetes version 1.22.
 
-As of Kubernetes v1.23, FlexVolumes have been deprecated in favor of the Container Storage Interface (CSI). Since our current application layer policy integration depends on FlexVolumes, it will not work on Kubernetes v1.23+.
```

Comment:
CSI was introduced a while back in K8s 1.13 (at least it moved into GA then). According to https://kubernetes-csi.github.io/docs/kubernetes-changelog.html the only feature change we need is the introduction of the `CSIDriver` object which was introduced in K8s 1.18. Since the version of Calico this will release with (v3.23) probably will not have K8s support that far back, we should be good without needing to call out any specific minimum versions.